### DISCUSSION TITLE
Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship. Check out our guide on how to contribute to open-source projects to learn how you can make a meaningful impact! Keep up the great work and you’ll be a perfect fit in no time!

### DISCUSSION THREAD

**Discussion by rithwik003** [community/community#51928] [GitHub Education]
Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship. Check out our guide on how to contribute to open-source projects to learn how you can make a meaningful impact! Keep up the great work and you’ll be a perfect fit in no time!

But i have made many contributions.

**Comment by Sooryak12** [comment]
did you figure it out ?
